src/openforms/prefill/contrib/stufbg/plugin.py

```diff
@@ -27,7 +27,14 @@
         "aoa.huisnummertoevoeging"
     ],
     FieldChoices.postcode: Target["verblijfsadres"]["aoa.postcode"],
-    FieldChoices.woonplaatsNaam: Target["verblijfsadres"]["wpl.woonplaatsNaam"],
+    # If the person lives in a different municipality, this field will be either in wpl.woonplaatsNaam
+    # or in aoa.woonplaatsWaarinGelegen > wpl.woonplaatsNaam
+    FieldChoices.woonplaatsNaam: {
+        "internal": Target["verblijfsadres"]["wpl.woonplaatsNaam"],
+        "external": Target["verblijfsadres"]["aoa.woonplaatsWaarinGelegen"][
+            "wpl.woonplaatsNaam"
+        ],
+    },
```

Comment:
This works but could be to magical (only documentation could clarify this).

Can we refactor it slightly so that we have 3 `FieldChoices`: `woonplaatsNaam`, `woonplaatsWaarinGelegenWoonplaatsNaam` and `woonplaatsNaamOrwoonplaatsWaarinGelegenWoonplaatsNaam`  (cry)?

Reply:
Then we can go for 1 :smile: Should I add a word in the docs about it?